### PR TITLE
Implement a calendar to nutrition screen

### PR DESCRIPTION
**Pull request by kokosaka**

### CODE REVIEW

File: screens/NutritionScreen.js [hunk 1]
```diff
@@ -1,23 +1,57 @@
...
+
+    const onDateChange = ({ date }) =>
+      this.setState({ ...this.state, date });
+
+
+    return (
+      <SafeAreaView style={styles.containerNutScreen}>
+        <View >
+          <Dates
+            onDatesChange={onDatesChange}
```
Comment (riz671) [inline screens/NutritionScreen.js:63]: Woah nice!

File: screens/NutritionScreen.js [hunk 2]
```diff
@@ -1,23 +1,57 @@
...
+import moment, { now } from 'moment';
 
-const NutritionScreen = (props) => {
-  return (
-    <SafeAreaView style={styles.containerNutScreen}>
-      <View style={styles.percentageContainerNutScreen}>
-        <TotalCaloriePercentage />
-      </View>
+export default class NutritionScreen extends Component {
+  state = {
```
Comment (riz671) [inline screens/NutritionScreen.js]: Is it possible to change to react hooks?

**Review by riz671 — APPROVED** [review]
It looks good to me. If possible, I'm wondering if we could refactor this into react-hooks in a future PR.

File: screens/NutritionScreen.js (hunk 2 above)
Comment (kokosaka) [reply screens/NutritionScreen.js]: refactored to react hooks

**Pull request merged**